enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -2876,6 +2882,13 @@ class AdjointGenerator
             }
           }
         }
+      if (looseTypeAnalysis) {
+        forwardModeInvertedPointerFallback(BO);
```

Comment:
this does not belong here right?

Reply:
Not strictly necessary rn, since I currently just use reverse mode. But I guess it doesn't hurt either, in case I run fwd mode later? Shall I remove it?

Reply:
No that's fine. I was for a moment confused about `forwardModeInvertedPointerFallback`, but that seems fine now.